### ISSUE TITLE
RPC-JSON API missing parts

### ISSUE THREAD

**Issue by shakibamoshiri** [SoftEtherVPN/SoftEtherVPN#1768]
### Prerequisites

* [X] Can you reproduce?
* [X] Are you running the latest version of SoftEtherVPN?

**SoftEther version:**
Both stable and DE version

### Description
It is about three months I am working on a CLI to manage SE servers using RPC-JSON. I would like to share my thoughts and issues I faced 

**first**
When we request to a SE server, the response does not contain the command name
here is a sample for `Test`
```json
{
  "result": {
    "Int64Value_u64": 0,
    "IntValue_u32": 0,
    "StrValue_str": "0",
    "UniStrValue_utf": ""
  },
  "jsonrpc": "2.0",
  "id": "rpc_call_id"
}
```

it could be better
```json
{
  "method": "Test",
  "result": {
    "Int64Value_u64": 0,
    "IntValue_u32": 0,
    "StrValue_str": "0",
    "UniStrValue_utf": ""
  },
  "jsonrpc": "2.0",
  "id": "rpc_call_id"
}
```

**second**
When there is an error (e.g " Object not found) the response is not robust enough 

sample
```json
{
  "error": {
    "code": 29,
    "message": "Error code 29: Object not found."
  },
  "jsonrpc": "2.0",
  "id": "rpc_call_id"
}
```

it could be better
```json
{
  "method": "GetUser",
  "result": {
    "error": true
    "code": 29,
    "message": "Error code 29: Object not found."
  },
  "jsonrpc": "2.0",
  "id": "rpc_call_id"
}
```

**third**
having extra dots "." in key names (e.g "Send.UnicastCount_u64").  
It makes it hard to parse the JSON response 

it could be
 - SendUnicastCount_u64
 - send_unicast_count_u64
or more standard you could make new object for each subject

instead of having this

```json
    "policy:Access_bool": true,
    "policy:ArpDhcpOnly_bool": false,
    "policy:AutoDisconnect_u32": 0,
    "policy:CheckIP_bool": false,
    "policy:CheckIPv6_bool": false,
    "policy:CheckMac_bool": false,
    "policy:DHCPFilter_bool": false,
    "policy:DHCPForce_bool": false,
    "policy:DHCPNoServer_bool": false,
    "policy:DHCPv6Filter_bool": false,
    "policy:DHCPv6NoServer_bool": false,
    "policy:FilterIPv4_bool": false,
    "policy:FilterIPv6_bool": false,
    "policy:FilterNonIP_bool": false,
    "policy:FixPassword_bool": false,
    "policy:MaxConnection_u32": 32,
    "policy:MaxDownload_u32": 0,
    "policy:MaxIP_u32": 0,
    "policy:MaxIPv6_u32": 0,
    "policy:MaxMac_u32": 0,
    "policy:MaxUpload_u32": 0,
    "policy:MonitorPort_bool": false,
    "policy:MultiLogins_u32": 1,
    "policy:NoBridge_bool": false,
    "policy:NoBroadcastLimiter_bool": false,
    "policy:NoIPv6DefaultRouterInRA_bool": false,
    "policy:NoIPv6DefaultRouterInRAWhenIPv6_bool": false,
    "policy:NoQoS_bool": false,
    "policy:NoRouting_bool": false,
    "policy:NoRoutingV6_bool": false,
    "policy:NoSavePassword_bool": false,
    "policy:NoServer_bool": false,
    "policy:NoServerV6_bool": false,
    "policy:PrivacyFilter_bool": false,
    "policy:RAFilter_bool": false,
    "policy:RSandRAFilter_bool": false,
    "policy:TimeOut_u32": 20,
    "policy:Ver3_bool": true,
    "policy:VLanId_u32": 0,
    "Realname_utf": "shakiba",
```

it could be

```json
  "result": {
    "AuthType_u32": 1,
    "CreatedTime_dt": "2023-01-29T19:28:48.325Z",
    "ExpireTime_dt": "2023-12-30T20:30:00.000Z",
    "GroupName_str": "",
    "HashedKey_bin": "H8N7rT8BH44q0nFXC9NlFxetGzQ=",
    "HubName_str": "vpn",
    "Name_str": "vpn",
    "Note_utf": "9000000000",
    "NtLmSecureHash_bin": "dmHLrLNggSwVTB2vSovo7g==",
    "NumLogin_u32": 29,
    "policy": {
      "Access_bool": true,
      "ArpDhcpOnly_bool": false,
      "AutoDisconnect_u32": 0,
      "CheckIP_bool": false,
      "CheckIPv6_bool": false,
      "CheckMac_bool": false,
      "DHCPFilter_bool": false,
      "DHCPForce_bool": false,
      "DHCPNoServer_bool": false,
      "DHCPv6Filter_bool": false,
      "DHCPv6NoServer_bool": false,
      "FilterIPv4_bool": false,
      "FilterIPv6_bool": false,
      "FilterNonIP_bool": false,
      "FixPassword_bool": false,
      "MaxConnection_u32": 32,
      "MaxDownload_u32": 0,
      "MaxIP_u32": 0,
      "MaxIPv6_u32": 0,
      "MaxMac_u32": 0,
      "MaxUpload_u32": 0,
      "MonitorPort_bool": false,
      "MultiLogins_u32": 1,
      "NoBridge_bool": false,
      "NoBroadcastLimiter_bool": false,
      "NoIPv6DefaultRouterInRA_bool": false,
      "NoIPv6DefaultRouterInRAWhenIPv6_bool": false,
      "NoQoS_bool": false,
      "NoRouting_bool": false,
      "NoRoutingV6_bool": false,
      "NoSavePassword_bool": false,
      "NoServer_bool": false,
      "NoServerV6_bool": false,
      "PrivacyFilter_bool": false,
      "RAFilter_bool": false,
      "RSandRAFilter_bool": false,
      "TimeOut_u32": 20,
      "Ver3_bool": true,
      "VLanId_u32": 0
    },
    "Realname_utf": "shakiba",
    "Recv.BroadcastBytes_u64": 3119084,
    "Recv.BroadcastCount_u64": 51031,
    "Recv.UnicastBytes_u64": 301498385,
    "Recv.UnicastCount_u64": 319339,
    "Send.BroadcastBytes_u64": 29712,
    "Send.BroadcastCount_u64": 258,
    "Send.UnicastBytes_u64": 25792932,
    "Send.UnicastCount_u64": 181268,
```

There are some bugs, but I keep them separate from this issue.
Regards

**Comment by chipitsine** [comment]
Wow
